BaseProject/Services/RollbarService.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  RollbarService.swift
+//  Istali
+//
+//  Created by Pablo Giorgi on 5/18/17.
+//  Copyright © 2017 Wolox. All rights reserved.
+//
+
+import Rollbar
+import Core
+import ReactiveSwift
+
+internal protocol RollbarServiceType {
```

Comment:
why do you need this? If you do, I'd write the {} in the same line as the declaration